cylc/flow/task_pool.py

```diff
@@ -2109,48 +2119,91 @@ def _get_flow_nums(
             for n in flow
         }
 
-    def _force_trigger(self, itask):
-        """Assumes task is in the pool"""
-        # TODO is this flag still needed, and consistent with "cylc set"?
+    def _force_trigger(self, itask: 'TaskProxy', on_resume: bool = False):
+        """Process a manually triggered task, ready for job submission.
+
+        Assumes the task is in the pool.
+
+        Triggering a queued task will:
+          - run it, regardless of queue limiting
+
+        Triggering an unqueued task will:
```

Comment:
(To me, "unqueued" implies that it was once queued and no longer is)
```suggestion
        Triggering an non-queued task will:
```